spec/system/jobseekers/jobseeker_applications_statuses_spec.rb

```diff
@@ -103,6 +103,10 @@
             click_link("Complete section")
           end
 
+          click_on "Add another job"
+          fill_in_current_role(start_year: jobseeker_profile.employments.first.ended_on.year.to_s, start_month: jobseeker_profile.employments.first.ended_on.month.to_s)
+          click_on "Save role"
+
```

Comment:
Changes in unrelated tests are filling in gaps in job applications so that they pass the new validation